lib/fluent/command/fluentd.rb

```diff
@@ -345,17 +345,6 @@
 exit 0 if early_exit
 
 if opts[:supervise]
-  if Fluent.windows?
-    if opts[:log_path] && opts[:log_path] != "-"
-      if opts[:log_rotate_age] || opts[:log_rotate_size]
-        require 'pathname'
-
-        log_path = Pathname(opts[:log_path]).sub_ext("-supervisor#{Pathname(opts[:log_path]).extname}").to_s
-        opts[:log_path] = log_path
```

Comment:
I wondered if this value is used for anything else, so I checked it.

This `log_path` value will be used in 2 other locations, but this change does not affect them.
So this looks fine.

Firstly, the config passed to ServerEngine has this value

* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/supervisor.rb#L474

However, ServerEnigne does not use this value and neither do ServerModule and WorkerModule.
So we can stop passing this `log_path` value to ServerEngine, but it has nothing to do with this issue.

Secondly, the `options` method uses this value.

* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/supervisor.rb#L727-L735

The `in_monitor_agent` plugin will use this.

* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/plugin/in_monitor_agent.rb#L403-L410 
* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/plugin/in_monitor_agent.rb#L399-L401
* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/plugin/in_monitor_agent.rb#L64-L83
* https://github.com/fluent/fluentd/blob/a3b1cdaa8c171f51e2b3d37e7f83b263243a236b/lib/fluent/plugin/in_monitor_agent.rb#L216-L217

We can get the `log_path` value by `api/config` or `api/config.json` of `in_monitor_agent`.

However, this path is only for workers.
We can't know supervisor's `log_path` since this is just one plugin's function.

The previous logic overwrote only the supervisor's `log_path`.
So this fix does not affect this plugin.

We can fix this to allow this plugin to display the actual `log_path` of each worker, but I think this should be separated from this issue as well.